vtr_flow/scripts/run_vtr_flow.pl

```diff
@@ -1687,3 +1711,74 @@ sub calculate_relaxed_W {
 
     return $relaxed_W;
 }
+
+sub cmp_full_vs_incr_STA {
+	# loop indices
+	my @indices = (0..6);
```

Comment:
I think using an array for the indices is unneccesary, since you have an array of the actual strings below.